src/nvme/mi-mctp.c

```diff
@@ -0,0 +1,212 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
+/*
+ * This file is part of libnvme.
+ * Copyright (c) 2021 Code Construct Pty Ltd
+ *
+ * Authors: Jeremy Kerr <jk@codeconstruct.com.au>
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <sys/uio.h>
+
+#include <ccan/endian/endian.h>
+
+#include "private.h"
+#include "log.h"
+#include "mi.h"
+
+#if 1
```

Comment:
Left over from debugging?

Reply:
Not really - more a marker for where we make this conditional on the new MCTP definitions being present in the usual toolchain headers.

Once we get things like `AF_MCTP`, `struct sockaddr_mtctp` etc present in those, we'll add an actual condition here.

If you prefer though, I can put that conditional in now - we're at a stage where we know what defs may be present in the kernel headers from actual releases.

Reply:
I just stumbled over it when quickly looking through the code. It looks odd without an explanation. I'd say if you can
add a proper conditional now, go for it :)

Reply:
OK, can do!